### PR TITLE
GRO-228: Updated Copy Text for ToS and Subscription checkboxes

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
This is to update the Copy Script for the Onboarding Sign Up form [GRO-228].

Past behavior:
Screen Shot 2021-04-21 at 12 00 59 PM

New behavior:
Screen Shot 2021-04-22 at 1 40 28 PM

[GRO-228]: https://artsyproduct.atlassian.net/browse/GRO-228

### CODE REVIEW

**Comment by The-Beez-Kneez** [comment]
Would like to confirm whether or not we wanted the blue text and hover underline enabled for these. That is what is shown in the ticket, but not sure if that is JIRA adding additional styling to Links in their system.

File: src/v2/Components/Authentication/TermsOfServiceCheckbox.tsx [hunk 1]
```diff
@@ -16,12 +16,31 @@ export const TermsOfServiceCheckbox = ({
     <StyledCheckbox {...{ checked: value, error, onChange, onBlur, name }}>
       <Serif color={color} size="3t" ml={0.5}>
         {"By checking this box, you consent to our "}
+        <Link
+          href="https://www.artsy.net/terms"
+          target="_blank"
+          color="blue100"
```
Comment (dzucconi) [inline src/v2/Components/Authentication/TermsOfServiceCheckbox.tsx:22]: I don't know if this is relevant here at all but if you like there's the `brand` color which is purple in v2 contexts and blue in v3 contexts.

Comment (The-Beez-Kneez) [reply src/v2/Components/Authentication/TermsOfServiceCheckbox.tsx:22]: Wait, so which is preferred purple or blue for links?

Comment (The-Beez-Kneez) [reply src/v2/Components/Authentication/TermsOfServiceCheckbox.tsx:22]: Sounds like the new v3 way is blue

Comment (The-Beez-Kneez) [reply src/v2/Components/Authentication/TermsOfServiceCheckbox.tsx]: Fixed after discussion in standup.

**Pull request merged**